samples/hocr/1.1/433934212_0017.html.yml

```diff
@@ -0,0 +1,8 @@
+engine: ocropy 1.0
+date: 2016-09-13
+engine-latest-commit: https://github.com/tmbdev/ocropy/tree/97ace5581b4c4deb95cfcd17b595b0ff1da36c25
```

Comment:
I don't know if such additional metadata are allowed. It should help to describe the used Ocropus version...